stakingModel/contracts/GoodReserveCDai.sol

```diff
@@ -208,6 +202,7 @@ contract GoodReserveCDai is DSMath, FeelessScheme, ActivePeriod {
      */
     function start() public onlyRegistered {
         addRights();
+        GoodDollar gooddollar = GoodDollar(address(avatar.nativeToken()));
```

Comment:
why do we need this if you read it dynamically eveyrwhere?

Reply:
we need to add the reserve as a minter
updated 53102ee50aa12ced8bb5ae3e39d40ea62ded976d